packetbeat/docs/packetbeat-options.asciidoc

```diff
@@ -122,6 +122,10 @@ packetbeat.interfaces.device: 0
 
 Specifying the index is especially useful on Windows where device names can be long.
 
+Alternatively the `default_route` device may be specified. This will set the capture
+device to be the device that is associated with the first default IPv4 route identified
+on Packetbeat start-up.
```

Comment:
I'm thinking we should explicitly state that it won't change the monitored device after start-up.

Reply:
Added. This was my intention with the current wording, but I guess it is a little subtle.